tempesta_fw/sock.c

```diff
@@ -433,7 +433,7 @@ ss_droplink(struct sock *sk)
 int
 __ss_close(struct sock *sk, int flags)
 {
-	if (unlikely(!sk))
+	if (unlikely(!(sk && ss_sock_live(sk))))
```

Comment:
Why can't we close for example half-open or half-closed sockets? I believe we should be able to close such sockets.